### PR TITLE
linked list code challenge

### PR DESCRIPTION
**Pull request by lhchandler4**

### CODE REVIEW

**Comment by lhchandler4** [comment]
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

File: challenges/src/main/java/Java/LinkedList/LinkedList.java [hunk 1]
```diff
@@ -50,4 +56,41 @@ public String prints() {
         return answer;
     }
 
+    //Insert Before (illegal argument exception)
+    public void insertBefore(int x, int data) {
```
Comment (evanslaton) [inline challenges/src/main/java/Java/LinkedList/LinkedList.java:60]: x isn't a particularly illuminating variable name, I highly recommend using variable names that are immediately meaningful (something like `valueToInsertBefore`)
